### ISSUE TITLE
Flask Issue

### ISSUE THREAD

**Issue by deeplearning987** [Prateek93a/DoodleAI#4]
Hello @Prateek93a

Still the same error. 

Unable to load the model.
<img width="957" alt="model-saving" src="https://user-images.githubusercontent.com/89828183/134128863-6a00a1fa-7cfb-4392-8a3a-c96982cb89ab.PNG">

![Screenshot (30)](https://user-images.githubusercontent.com/89828183/134128938-00c223e5-0767-4c5b-906c-ae8147fb0d09.png)

**Comment by Prateek93a** [comment]
That is not how you put comments in javascript. Use // instead of #.

**Comment by deeplearning987** [comment]
I changed it replaced // instead of #.
The Issue remains same.
Please help.

![Screenshot (33)](https://user-images.githubusercontent.com/89828183/134143416-cc991453-74a5-4927-b961-f3904df8a5a6.png)

**Comment by Prateek93a** [comment]
Show the image of browser console

**Comment by deeplearning987** [comment]
<img width="659" alt="terminal" src="https://user-images.githubusercontent.com/89828183/134144460-55c63474-7f0f-454b-9fc5-4e2bfd8a787c.PNG">

Can you check it once

**Comment by Prateek93a** [comment]
This is not browser console. You can google about how to access the browser console window. I have checked the repo locally and it is working fine. Check your browser console and if you find some issue, try googling it and solving the problem. Make sure you understand the code given in index.js file.

**Comment by deeplearning987** [comment]
<img width="958" alt="browser console" src="https://user-images.githubusercontent.com/89828183/134146771-e24a48e4-6a2c-4bd5-b131-b3203ff9172e.PNG">


I misunderstand about console. can you check browser console I've shared.

**Comment by Prateek93a** [comment]
Clear the browser cache and reload. Do Ctrl+Shift+r

**Comment by deeplearning987** [comment]
ok

**Comment by Prateek93a** [comment]
Issue solved?

**Comment by deeplearning987** [comment]
No not yet.
<img width="958" alt="browser console" src="https://user-images.githubusercontent.com/89828183/134148094-e541f5b2-15f8-4443-bf6f-6399ca60876a.PNG">

**Comment by deeplearning987** [comment]
<img width="957" alt="browser issue" src="https://user-images.githubusercontent.com/89828183/134148133-47c52314-31c7-4dc7-822e-7e7ef8905e5e.PNG">


The Latest Issue.

**Comment by Prateek93a** [comment]
See the url in the console. It is not 127.0.0.1:5000/static/model/onnx_model.onnx. This means, either you haven't saved the changes I asked you to make in js file, or you did not restart the flask server, or you haven't cleared the cache. I think from here, you will be able to handle the issue. Most probably it is cache problem. But you can figure it out.

**Comment by deeplearning987** [comment]
I've made the changes in js file as you've mentioned.

await sess.loadModel( "http://127.0.0.1:5000/static/model/onnx_model.onnx");

But it is still running on 127.0.0.1:5000

**Comment by Prateek93a** [comment]
Make sure you have saved the changes. Restart the flask server. Now clear the cache or run on different browser.

**Comment by deeplearning987** [comment]
Hello @Prateek93a 
For the LoadModel we need to mention where the load model has to be present in the local environment.
we did that one.

The issue is like below.

session.ts:61 Fetch API cannot load file:///C:/UsersAbDownloadsDoodleAI-mastersDoodleAI-masterstaticmodelonnx_model.onnx. URL scheme "file" is not supported.

Actually the trained model is nothing but a file.When to try to do that from local machine it will act like a file rather than API.

In the index.html/layout.html we're unable to locate the LOAD BUTTON AND CLEAR BUTTON.

In the index.js we changed this path of local folder where the onnx file is present.
await sess.loadModel("\DoodleAI-master\static\model\onnx_model.onnx");
